app/package.json

```diff
@@ -11,6 +11,7 @@
     "@rjsf/core": "^3.2.1",
     "@rjsf/semantic-ui": "^3.2.1",
     "next": "latest",
+    "next-session": "^3.4.0",
     "react": "^17.0.2",
```

Comment:
Version controlled this, it seems to be broken on the lates version for serversideprops